platform/platform-app/pom.xml

```diff
@@ -455,7 +458,6 @@
         <cxf.xmlresolver.bundle.version>1.2_5</cxf.xmlresolver.bundle.version>
         <cxf.xerces.bundle.version>2.11.0_1</cxf.xerces.bundle.version>
         <cxf.bcel.bundle.version>5.2_4</cxf.bcel.bundle.version>
-        <cxf.xalan.bundle.version>2.7.2_3</cxf.xalan.bundle.version>
```

Comment:
I fixed this per @emmberk . Pattern is to declare in the 'must match' section of the root pom, then redeclare using the root declaration in platform-app so that block matches as closely as possible to the CXF block it is copied from.